ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -5,6 +5,45 @@ set -o errexit
 set -o pipefail
 set -x
 
+run_scorecard() {
+        local retries_max=3
+        local attempt_num=1
+        until [[ $(jq . $2) ]]
```

Comment:
nit: can you double quote `$2` to prevent globbing and word splitting (we get warnings if we don't do that)